### ISSUE TITLE
Uploading files with extension JPG in capital letters leads to error from File upload field

### ISSUE THREAD

**Issue by alulay** [pods-framework/pods#5975]
Uploading files with extension JPG in capital letters leads to error message: 'Error: File type not allowed, please use one of the following: .jpg, .jpeg, .png, .gif'.
After renaming the same file with extension jpg in lower case the upload is successful.
The upload is for an Advanced Content Type pod. The 'Images' field has the type 'File / Image / Video', the Restrict Field Types field has the value 'Images (jpg, jpeg, png, gif)', the List Style field has the value 'Tiles'. 

## Testing instructions

1. Selection of the pod of type 'Advanced Content Type' in the Wordpress backend for which an image is to be uploaded.
2. Select Add Button for files and select jpg-file 'image.JPG' 
3. See error message 'Error: File type not allowed, please use one of the following: .jpg, .jpeg, .png, .gif'.

## Expected behavior

The upload should work regardless of whether the extension is uppercase or lowercase.

## Screenshots / screencast

![grafik](https://user-images.githubusercontent.com/50745099/111500914-c9e9ca00-8744-11eb-887e-9e235bff4065.png)

## Pods version

2.7.26

## Other super helpful information

With Pods v2.7.23 the error does not occur.

<details>
<summary>WordPress Environment</summary>
  
<summary>WordPress Environment</summary>
Website-Zustand
Gut
Status
Bericht
Zustand der Website

Der Test zum Website-Zustand zeigt wichtige Informationen zu deiner WordPress-Konfiguration und zu Elementen, die deine Aufmerksamkeit erfordern.
1 empfohlene Verbesserung
Auf deiner Website läuft eine ältere Version von PHP (7.3.27) Leistung

</details>

<details>
<summary>Pods Package Export</summary>

```json
Copy and paste your Pods Admin > Migrate: Packages > Export and paste the code over this line.
```

</details>

**Comment by sc0ttkclark** [comment]
Should be an easy fix for a 'good first PR' here.

Within the `admin_ajax_upload()` method in `classes/fields/file.php` there is a line that does a conditional check:

`if ( $limit_type === $file_info['extension'] ) {`

That check is case sensitive, so it could just force both sides to use lowercase for the comparison.

```php
	if ( isset( $file_info['extension'] ) ) {
		// Enforce lowercase for the extension checking.
		$file_info['extension'] = strtolower( $file_info['extension'] );

		foreach ( $limit_types as $limit_type ) {
			$limit_type = strtolower( trim( $limit_type, ' .' ) );

			if ( $limit_type === $file_info['extension'] ) {
				$ok = true;

				break;
			}
		}
	}
```

**Comment by sc0ttkclark** [comment]
Fixed in #5996
